src/main/sql/example_data.sql

```diff
@@ -1,8 +1,23 @@
 --
--- Example Client, please sync to the attribute values in the addon-administration.properties.
--- Please update all queries in this file to your needs!
--- Have to be imported in the database of the auth-server, before you deploy the addon-administration!
+-- Example Client, please sync to the attribute values in the
+-- addon-administration.properties. Have to be imported in the database of the
+-- auth-server, before you deploy the addon-administration!
 --
+
+INSERT INTO osiam_client (internal_id, accesstokenvalidityseconds, client_secret, expiry,
+                          id, implicit_approval, redirect_uri, refreshtokenvalidityseconds,
+                          validityinseconds)
+VALUES (20, 28800, 'super-secret', NULL,
+        'addon-administration-client', TRUE, 'http://localhost:8080/addon-administration', 86400,
+        0);
+
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'GET');
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'POST');
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'PUT');
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'PATCH');
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'DELETE');
+INSERT INTO osiam_client_grants (id, grants) VALUES (20, 'authorization_code');
+
 -- resource server: example group 'admin' for the addon-administration
 
 INSERT INTO scim_meta (id, created, lastmodified, location, resourcetype, version)
```

Comment:
we should make use of the [postgres](http://www.postgresql.org/docs/9.3/static/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT) and [mysql](https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_current-date) CURRENT_DATE function